bin/BuildPackages.sh

```diff
@@ -279,8 +248,8 @@ build_one_package() {
   ) || build_fail
 }
 
-date >> "$LOGDIR/$FAILPKGFILE.log"
-for PKG in "${PACKAGES[@]}"
+date >> "$LOGDIR/fail.log"
+for PKG in ${PACKAGES}
```

Comment:
BTW, I think we currently do not restrict package *names* in any way (of course the package name does not have to be related to the package directory name at all, it's just a useful convention). We might want to revise that, too, and e.g. insist that it is a valid identifier (as in `IsValidIdentifier`). I kind of thought that's necessary for our namespaces anyway, but `ValidatePackageInfo` doesn't check it, it seems? @alex-konovalov ?